src/data.js

```diff
@@ -139,3 +139,40 @@ Namespace.ensureNamespace = function(namespace, expires) {
     });
   });
 };
+
+/**Delete expired entries */
+Namespace.expireEntries = function(parent, continuationToken) {
+  var that = this;
+
+  return that.query({
+    parent: parent,
+  },
+  {
+    limit:         500,
+    continuation:   continuationToken,
+  }).then(async (data) => {
+    var dataLength = data.length;
+    // Stop recursion if the namespace is an indexed task
+    if (dataLength == 0) {
+      return;
+    }
```

Comment:
I think this conditional is unnecessary -- the loop below will not execute if `dataLength` is zero.  Also, sometimes Azure returns no results but a continuationToken (it has something to do with how the data is stored on their servers), in which case the `if (data.continuation)` below would evaluate as true.

Reply:
OK. I removed that .